server/__tests__/unit/services/Keys.js

```diff
@@ -1,67 +1,85 @@
 const { createKey, fetchKeysByuserid, isAPIKeyPresent, destroyKey } = require("../../../services");
-const { KeyCollection } = require("../../../utils/firestore");
-const { Keys } = require("../../../models");
-const { Timestamp } = require("firebase-admin/firestore");
-const { v4 } = require("uuid");
+const Keys = require("../../../models/Keys");
 const { mockKeys } = require("../../../utils/mocks/Keys");
+const mongoose = require("mongoose");
+const { MongoMemoryServer } = require("mongodb-memory-server");
 
-jest.mock("uuid", () => ({
-  v4: jest.fn(() => mockKeys[0].keyId),
-}));
+beforeAll(async () => {
+  const mongoServer = await MongoMemoryServer.create();
+  const mongo_uri = mongoServer.getUri();
+  await mongoose.connect(mongo_uri);
+});
 
-afterEach(() => {
-  jest.restoreAllMocks();
+afterAll(async () => {
```

Comment:
Is this required here ? Some test files doesn't have `afterAll` and `beforeAll`.